docs/developer/creating-test-environment.md

```diff
@@ -18,7 +18,7 @@ You can choose between either a single-cluster or a two-cluster setup.
 
 ### Create module operator and bundle module
 
-To bundle your module image and operator, please refer to the detailed information inside [template-operator docs](../../samples/template-operator/README.md#bundling-and-installation).
+To bundle your module image and operator, please refer to the detailed information inside [template-operator docs](../../../template-operator/README.md#bundling-and-installation).
```

Comment:
```suggestion
To bundle your module image and operator, please refer to the detailed information inside [template-operator docs](https://github.com/kyma-project/template-operator/blob/main/README.md#bundling-and-installation).
```

Reply:
why is this change necessary? this will link to an unbounded file in main, instead of a relative path.

Reply:
The link `../../../template-operator/README.md#bundling-and-installation` goes outside of this repository. Thus, with this link you assume the user has already cloned the template-operator repository into the parent directory of the cloned lifecycle-manager repository.
If the user did not clone it, it does not work the link